paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -170,19 +182,32 @@ def paasta_start_or_stop(args, desired_state):
         print msg
         return 1
 
-    if 'refs/heads/paasta-%s' % service_config.get_deploy_group() not in remote_refs:
+    invalid_deploy_groups = []
+    for cluster in clusters:
+        service_config = get_instance_config(
+            service=service,
+            cluster=cluster,
+            instance=instance,
+            soa_dir=soa_dir,
+            load_deployments=False,
+        )
+
+        if 'refs/heads/paasta-%s' % service_config.get_deploy_group() not in remote_refs:
+            invalid_deploy_groups.append(service_config.get_deploy_group())
+        else:
+            force_bounce = utils.format_timestamp(datetime.datetime.utcnow())
+            issue_state_change_for_service(
+                service_config=service_config,
+                force_bounce=force_bounce,
+                desired_state=desired_state,
+            )
+
+    if invalid_deploy_groups:
         print "No branches found for %s in %s." % \
-            (service_config.get_deploy_group(), remote_refs)
+            (", ".join(invalid_deploy_groups), remote_refs)
         print "Has it been deployed there yet?"
```

Comment:
Now that we might output multiple deploy groups here it might make sense to use quantity-neutral language here.
I'm all for `print "Has %s been deployed there yet?" % service`